src/SipHash.chpl

```diff
@@ -55,6 +56,17 @@ module SipHash {
             (p[7]: uint(64) << 56));
   }
 
+  private inline proc XTO64_LE(in x: ?t) {
+    var y: uint(64);
+    if isSubtype(t, c_ptr) {
+      c_memcpy(c_ptrTo(y), x, 8);
```

Comment:
Should this always be copying 8-bytes or should it be `c_sizeof(t)`?

Reply:
Oh, that's how you say that! It's been so long since I've done any serious C programming... 

To answer your question, yes, in all the use cases intended in this code, this line should always copy 8 bytes, but it probably is a good idea to future-proof with `c_sizeof(t)`.